client/src/app/pages/messages/messages.component.html

```diff
@@ -1 +1,13 @@
-<p>messages works!</p>
+<app-top-navbar [leftItem]="'logo'" [pageTitle]="'Chats'"></app-top-navbar>
+<div class="top-navbar-bottom-navbar-exclusive">
+    <div *ngFor="let chat of chats" [routerLink]="['/chat', chat.id]" class="chat-card">
```

Comment:
You could have used a <ul>, or a <mat-list> here, you should consider doing so after the migration

Reply:
You mean instead of a div? I was using one before but it gave some css problems.

Reply:
Yes, we should check those out, weird